assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -153,7 +153,7 @@ const Cart = ( {
 	};
 
 	const totalsCurrency = getCurrencyFromPriceResponse( cartTotals );
-	const totalRowsConfig = getTotalRowsConfig( cartTotals );
+	const totalRowsConfig = getTotalRowsConfig();
```

Comment:
Great to tidy this up! This came up in a recent PR (#1561) - the function got moved inside instead of explicit params, and this got missed along the way. Another option for fixing this would be moving back out and reinstating param back :)